pallets/users/src/lib.rs

```diff
@@ -32,60 +35,83 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn something)]
-
+	// define pallet storage abilities up top and create function definitions inside
 	pub type Something<T> = StorageValue<_, u32>;
```

Comment:
Need to update this. The rest is good.

Reply:
Updated. That part is boilerplate so I simply removed it.
Fn, User only contains `rank_points`